### DISCUSSION TITLE
Global Variable

### DISCUSSION THREAD

**Discussion by jojogib** [RomanKubiak/ctrlr#536] [Q&A]
Hey, 
[globalvar2.zip](https://github.com/RomanKubiak/ctrlr/files/10530304/globalvar2.zip)

f0 42 00 2a 48 00 MM NN 03 00 01 00 LS MS f7
f0 42 00 2a 48 00 MM NN 03 00 02 00 LS MS f7

MM = Channel
NN= Part

I want to change MM and NN for the partnumber with expression to evaluate.
1 Combobox with the NN Valuesand one with MM to change this value on all selcted modualtors.
I tried the demopanel with complex expressions but no success. I can change NN but then LS MS dont work as it should anymore.
Thanks for your help

**Comment by dnaldoog** [comment]
Do you mean a global variable like this?

![globvarfield](https://user-images.githubusercontent.com/76831683/215406929-f9c343d7-0c29-41f4-a82c-a351a63eb19b.png)

Set that field (in one of your combos) to a number and access it in an another modulator as `k1` (in this case)

`f0 42 00 2a 48 00 00 k1 03 00 01 k2 ls ms f7`

[global variable in sysex_1_0.zip](https://github.com/RomanKubiak/ctrlr/files/10533198/global.variable.in.sysex_1_0.zip)
